### DISCUSSION TITLE
megaTinyCore - burn bootloader error

### DISCUSSION THREAD

**Discussion by DanSkareda** [SpenceKonde/megaTinyCore#517] [Library discussion]
Hi. I have a clean instalation Arduino 1.8.15. I have installed only megaTinyCore (from *.json web address) and python3-3.7.2.post1-embed-win32v2a.zip (according to the instructions to python3). When I want to burn the bootloader for attiny1614, the burn ends with an error:

Traceback (most recent call last):
  File "C:\Users\radek\AppData\Local\Arduino15\packages\megaTinyCore\hardware\megaavr\2.4.0/tools/prog.py", line 282, in <module>
    main()
  File "C:\Users\radek\AppData\Local\Arduino15\packages\megaTinyCore\hardware\megaavr\2.4.0/tools/prog.py", line 112, in main
    fuse_offset, fuse_val = fuse_str.split(":")
ValueError: not enough values to unpack (expected 2, got 1)

Do you know what's wrong? I also tried it in last version arduino beta 2.0 and there is the same error. I programming with pyUPDI SLOW. Sketches I upload without errors. The same error when burning the bootloader is also when I programming attiny3216.

**Comment by DanSkareda** [comment]
We did it. But very difficult. I programmed the bootloader only with the megaTinyCore library version 2.4.2 in the <hardware> folder in my sketchbook. But if I then want to load the program, I have to delete the library in the <hardware> folder and I can only load the program via megaTinyCore version 2.4.0 (which is in the system via the *.json address). WTF??? It's all weird. Is it bugs in the library or what's going on?

**Comment by SpenceKonde** [comment]
Thi issue will be dcorrected in aq boqrd kmanager release within an hour.

Duplicate of https://github.com/SpenceKonde/megaTinyCore/issues/506

**Comment by SpenceKonde** [comment]
Thanks for erports I was awae of tje bug shortly after the release of 2.4.0; This issue is resolved in 2.4.1 and later (2.4.1 was pulled because of other issues) . Please use 2.4.2 of megaTinyCore,and everyrthing should work.

**Comment by SpenceKonde** [comment]
Please use released version of 2.4.2 from board manager

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***
